utils/manifest-utils/tests/certificate_store.rs

```diff
@@ -17,49 +18,49 @@ static TEST_RESOURCES: TestResources = TestResources {
     &[],
     &[],
     &[],
-    &[];
+    &["cert-permissions.json"];
     "Does not fail when listing empty store"
 )]
 #[test_case(
     &["foo_ca.cert.pem"],
     &[],
     &["c128af8c"],
-    &["foo_ca.cert.pem"]; 
+    &["foo_ca.cert.pem", "cert-permissions.json"]; 
     "Can load one certificate"
 )]
 #[test_case(
     &["foo_ca.cert.pem", "foo_inter.cert.pem"],
     &["c128af8c", "4e0df976"],
     &[],
-    &[];
+    &["cert-permissions.json"];
     "Can remove all certificates"
 )]
 #[test_case(
     &["foo_ca-chain.cert.pem"],
     &[],
     &["4e0df976", "c128af8c"],
-    &["foo_ca-chain.cert.pem"]; 
+    &["foo_ca-chain.cert.pem", "cert-permissions.json"]; 
     "Load keychain loads two certificates and stores them in received form (single keychain file)"
 )]
 #[test_case(
     &["foo_ca-chain.cert.pem"],
     &["c128af8c"],
     &["4e0df976"],
-    &["foo_ca-chain.cert.4e0df976.pem"]; 
+    &["foo_ca-chain.cert.4e0df976.pem", "cert-permissions.json"]; 
```

Comment:
Added here these "cert-permissions.json" everywhere without checking its content because this test has been written before `agent/provider/tests/keystore_cli.rs` and is now a bit redundant. I left it because it checks if certificates in keystore are still saved correctly (correct names).